instrument/config.go

```diff
@@ -58,6 +58,9 @@ type MetricsConfiguration struct {
 
 	// Extended metrics type.
 	ExtendedMetrics *ExtendedMetricsType `yaml:"extended"`
+
+	// Metric sanitization type.
+	MetricSanitization *MetricSanitizationType `yaml:"sanitization"`
```

Comment:
nit: just call this `Sanitization`. This is a metric configuration object so don't think we need another `metric` in the name.